### PR TITLE
Added noindex meta tags to previous versions of the documentation. 

### PR DESCRIPTION
**Pull request by goodlux**
Fixes issue with old versions of the documentation appearing in google search results. 

See PyTorch #3428: https://github.com/pytorch/pytorch/issues/3428

### CODE REVIEW

File: _layouts/default.html [hunk 1]
```diff
@@ -3,6 +3,8 @@
 <head>
     <link rel="shortcut icon" type="image/x-icon" href="favicon.ico">
     <title>{{ page.title }}</title>
+
+    <meta property="og:url" content="http://pytorch.org"/>
```
Comment (zou3519) [inline _layouts/default.html]: what is og:url? Should we use https here? Alternative, does `content="/"` work?

**Comment by goodlux** [comment]
The og (open graph) meta tags are Facebook standard metadata for sharing
links...they instruct applications like messenger or Facebook feed what url
to us, which image to display, proper meta text,  etc. There is another pr
on this, still open I believe.

**Comment by zou3519** [comment]
Cool, thanks for the explanation. @goodlux Could you change the content= link to use https instead of http? I don't know if it matters, but erring on the side of caution.

I looked at the meta robots noindex changes and they look good.

**Comment by goodlux** [comment]
done!

**Comment by JoelMarcey** [comment]
Updated site has been developed. Thank you, however, for your PR.

**Pull request closed without merging**